### ISSUE TITLE
tradfri signal enhancer supported device?

### ISSUE THREAD

**Issue by trekker25** [Koenkk/zigbee2mqtt#832]
Is the Tradfri Signal enhancer automatically supported as it's only a router?

https://www.ikea.com/de/de/catalog/products/10400408/

I was planning to buy 3-4 CC2531 and put router firmware on it, but this also seems a good solution if it's supported?

Because it's a router, it should be supported i think?

**Comment by danpowell88** [comment]
it would still need to be paired to the network, so if its not already in supported devices itd need to be added

**Comment by trekker25** [comment]
until now i see it's only available in Ikea Germany in store not in Netherlands or online to be ordered.

So have to wait for that?!?

**Comment by danpowell88** [comment]
someone will need access to one before it can be configured to work

**Comment by trekker25** [comment]
I see it's first available from 1th Februari in Ikea Dusseldorf, so will have to wait some time :)

**Comment by Koenkk** [comment]
Support can be added when somebody has this device, as this device does nothing more than routing, I think there is nothing to support :)

Note that a device **does not** have to be supported by zigbee2mqtt to serve as a router! When joins the zigbee network, it will automatically start doing this.

**Comment by rgruebel** [comment]
Will this router also work with the Xiaomi sensors? My Xiaomi devices cannot connect to my osram routers. I wonder if this is an osram or xiaomi problem?

**Comment by Koenkk** [comment]
@rgruebel sometimes Xiaomi sensors refuse to move to another router, repairing while having it close to the router sometimes solves the problem.

**Comment by boojew** [comment]
Seems like it will be available here in Canada around the 2nd week of Feb. I'll see if I can pick one up.

**Comment by Pockrandt** [comment]
if someone gets one please verify if this from the manual is correct:
> Max total load: 1000 mA

**Comment by nanosonde** [comment]
I have bought a repeater yesterday at Ikea Duesseldorf/Germany.

Of course I get: " Device with modelID 'TRADFRI signal repeater' is not supported."
It shows up as " unkown - undefined unknown (Router)"

There is a small whole in the housing of the repeater with a push-button which I kept pressing to see a reaction at all.

**Comment by Koenkk** [comment]
Device is supported now in the dev branch. (actually there is nothing to support but it just doesn't show as an unsupported device now).

**Comment by nanosonde** [comment]
I just want to mention that at least for me it is not clear what the correct procedure is to join it to the PAN.
I just pressed the button a couple of times and it SOMEHOW worked.

**Comment by nanosonde** [comment]
What is the "normal" procedure to join a pure router to the PAN? Does it need be paired too?
What about this touchlink stuff? Might it be that the router needs to placed close to the coordinator to get paired?

**Comment by Koenkk** [comment]
@nanosonde I expect the pairing procedure to be the same as of a TRADFRI bulb (factory reset it while having it close to the coordinator).

**Comment by nanosonde** [comment]
I restarted testing with the latest dev branch and the signal repeater as the only device.
Coordinator is CC2530 with  Coordinator firmware version: '20190109'.
The coordinator is located in a different room. No devices paired so far.

I was able to pair the Tradfri repeater by just pushing the button with a paperclip for at 5 seconds (as described in the manual: factory reset). 
It was NOT located close to the coordinator. It took about 30s and then it showed up in zigbee2mqtt.
